monitor/views.py

```diff
@@ -97,11 +96,31 @@ def monitor_home(request):
 
 
 def queries_stats_ajax(request):
-    req = urllib2.Request("http://mtg-logserver.s.upf.edu/graylog/api/search/universal/relative/terms?query=%2A&range=1209600&filter=streams%3A531051bee4b0f1248696785a&field=query")
-    base64string = base64.b64encode('%s:%s' % (settings.GRAYLOG_USERNAME,
-        settings.GRAYLOG_PASSWORD))
-    req.add_header("Authorization", "Basic %s" % base64string)
-    return JsonResponse(json.load(urllib2.urlopen(req)))
+    params = {
+        'query': ':',
+        'range': 14 * 60 * 60 * 24,
+        'filter': 'streams:531051bee4b0f1248696785a',
```

Comment:
I think it would be best to take out that string "API....." and put it in the preferences....